cli/internal/cmds/neosync/sync/sync.go

```diff
@@ -297,7 +297,9 @@ func (c *clisync) configureAndRunSync() error {
 			case <-ctx.Done():
 				return
 			case <-stopChan:
+				c.logger.Error("Sync Failed.")
 				cancel()
+				os.Exit(0)
```

Comment:
interesting. I guess because we arent cancelling the outer context. Should probably look into how we can do that so that the entire program can exit.

Also if this is an error should `os.Exit(1)`